### PR TITLE
Nn 3367 key worker settings update

### PR DESCRIPTION
**Pull request by jsrobertson**
image

With error message:
image

### CODE REVIEW

File: backend/utils.js [hunk 1]
```diff
@@ -159,6 +159,12 @@ const ensureIsArray = (data) => (Array.isArray(data) ? data : [data])
 
 const maybePluralise = (noun, count, suffix = 's') => `${count} ${noun}${count !== 1 ? suffix : ''}`
 
+const possessive = (string) => {
+  if (!string) return ''
+
+  return `${string}${string.toLowerCase().endsWith('s') ? '’' : '’s'}`
```
Comment (sp-watson) [inline backend/utils.js:165]: I have to ask - are you sure this is the correct apostrophy? I have seen Jamie use an html encoding to get the right one before.

File: views/manageKeyWorkerSettings.njk [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+          titleText: "There is a problem",
+          errorList: errors,
+          attributes: { "data-test": "form-errors" }
+        }) }}
+      {% endif %}
+
+      <h1 class="govuk-heading-l">{{ title }}</h1>
+
+      <form method="POST">
+        {{ govukRadios({
```
Comment (sp-watson) [inline views/manageKeyWorkerSettings.njk:37]: Does this need that hidden csrf token?

**Review by sp-watson — APPROVED** [review]

File: backend/utils.js (hunk 1 above)
Comment (jsrobertson) [reply backend/utils.js:165]: It's ok in this instance to use the character and is easier to read.  It's the same function copied over from PSH.

File: views/manageKeyWorkerSettings.njk (hunk 2 above)
Comment (jsrobertson) [reply views/manageKeyWorkerSettings.njk:37]: It's a good point, and probably yes, but it's something we've not introduced in this app yet.  Will raise a separate story to implement it.

**Pull request merged**